sklearn/pipeline.py

```diff
@@ -147,6 +149,23 @@ def __init__(self, steps, *, memory=None, verbose=False):
         self.memory = memory
         self.verbose = verbose
 
+    def set_output(self, transform=None):
+        """Set output container.
```

Comment:
I think that we should also mentioned that we mutate the original estimators. Here it is less surprising because the pipeline already mutate `steps` inplace :)